package-lock.json

```diff
@@ -1995,7 +1995,7 @@
         },
         "lodash": {
           "version": "4.17.4",
-          "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.4.tgz",
+          "resolved": "http://registry.npmjs.org/lodash/-/lodash-4.17.4.tgz",
```

Comment:
Some of these changes don't seem right, moving away from https? Did you perhaps use `npm config set registry http://...` when switching from internal registry?

Reply:
I don't think so 🤔 I've seen similar changes in the past, not sure why this happens.